api/server.py

```diff
@@ -14,6 +14,8 @@
 import datetime
 import os
 from seed_db import *
+import config
```

Comment:
I don't think we need this line - let's remove it so it doesn't get merged in :)